packages/playground/src/calculator/pricing_calculator.vue

```diff
@@ -153,28 +153,43 @@
           <VRow class="text-center text-body-1 text-black" v-else-if="valid">
             <VCol cols="6">
               <div
-                class="rounded pa-4 h-100 d-flex justify-center align-center"
-                :style="{ background: computePackageColor(priceTask.data?.dedicatedPackage.package) }"
+                class="rounded pa-4 discount"
+                :style="{
+                  background: computePackageColor(priceTask.data?.dedicatedPackage.package),
+                }"
               >
                 <p>
-                  Cost of reserving a <strong v-text="'Dedicated Node'" /> of the same specifications<br />
+                  Cost of reserving a <strong v-text="'Dedicated Node '" />
                   <strong v-text="'Loading...'" v-if="priceTask.loading" />
-                  <strong v-else v-text="dedicatedPriceUSD + ' USD/month, ' + dedicatedPriceTFT + ' TFT/month'" />. A
-                  user can reserve an entire node then use it exclusively to deploy solutions
+                  <strong v-else v-text="dedicatedPriceUSD + ' USD/month, ' + dedicatedPriceTFT + ' TFT/month.'" />
                 </p>
+                <section class="card" v-if="priceTask.data?.dedicatedPackage.package !== 'gold'">
+                  <p class="card-info">
+                    <b>Too expensive?</b> can upgrade to <b>Gold package</b> to get discount up to 60% when you fund
+                    your wallet with <b>{{ Math.ceil(114.473 * 18) }}</b> TFT
+                  </p>
+                </section>
               </div>
             </VCol>
             <VCol cols="6">
               <div
-                class="rounded pa-4 h-100 d-flex justify-center align-center"
-                :style="{ background: computePackageColor(priceTask.data?.sharedPackage.package) }"
+                class="rounded pa-4 h-100 discount"
+                :style="{
+                  background: computePackageColor(priceTask.data?.sharedPackage.package),
+                }"
               >
                 <p>
-                  Cost of reservation on a <strong v-text="'Shared Node'" /><br />
+                  Cost of reservation on a
+                  <strong v-text="'Shared Node '" />
                   <strong v-text="'Loading...'" v-if="priceTask.loading" />
-                  <strong v-else v-text="sharedPriceUSD + ' USD/month, ' + sharedPriceTFT + ' TFT/month'" />. Shared
-                  Nodes allow several users to host various workloads on a single node
+                  <strong v-else v-text="sharedPriceUSD + ' USD/month, ' + sharedPriceTFT + ' TFT/month.'" />
                 </p>
+                <section class="card" v-if="priceTask.data?.sharedPackage.package !== 'gold'">
+                  <p class="card-info">
+                    <b>Too expensive?</b> can upgrade to <b>Gold package</b> to get discount up to 60% when you fund
+                    your wallet with <b>{{ Math.ceil(228.947 * 18) }}</b> TFT
```

Comment:
I propose that we move these balance values to a `variable` instead of having them in the code.